schemas/dab.draft.schema.json

```diff
@@ -693,6 +693,166 @@
         }
       }
     },
+    "autoentities": {
+      "type": "object",
+      "description": "Auto-entity definitions for pattern matching",
+      "patternProperties": {
+        "^.*$": {
+          "type": "object",
+          "additionalProperties": false,
```

Comment:
how can additionalProperties be false? Instead, it should be the parent of properties like:

``` 
   "additionalProperties": {
    "type": "object",
    "properties": {
      "patterns": {
```